dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/notification/NotificationPagingParam.java

```diff
@@ -37,7 +37,7 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
-public class BaseNotificationParam {
+public class NotificationPagingParam {
```

Comment:
Is this class needed at all?

Reply:
i think so. Providing paging information to `ProgramNotificationInstanceParam`, `ProgramNotificationTemplateOperationParams`

Reply:
Maybe we can re-evaluate this after the pagination in store is completed as well